extras/qpsclient.py

```diff
@@ -41,7 +41,7 @@ def start_requests(self):
 
         slots = int(self.slots)
         if slots > 1:
-            urls = [url.replace('localhost', '127.0.0.%d' % (x + 1)) for x in xrange(slots)]
+            urls = [url.replace('localhost', '127.0.0.%d' % (x + 1)) for x in range(slots)]
```

Comment:
or may be `from six.moves import xrange` to get the generator version in both versions.